app.js

```diff
@@ -826,7 +826,9 @@ define(function(require) {
 				function(waterfallCallback) {
 					self.createUser(data.user,
 						function(userData) {
-							data.user.id = _.get(userData, 'data.id', '');
+							var userId = _.get(userData, 'data.id', '');
+							data.user.id = userId;
+							data.vmbox.owner_id = userId;
```

Comment:
While this works, the `createUser` handler unnecessarily has to "know" about vmboxes.

Since there is a dedicated request handler to create vmboxes, couldn't this be moved there, to collocate prop setting next to where it is actually used?